docker-compose.yml

```diff
@@ -1,56 +1,60 @@
 version: "3.9"
 services:
-  dashboard:
-    image: louislam/uptime-kuma:1
-    volumes:
-      - /var/run/docker.sock:/var/run/docker.sock
-      - ./uptime-kuma-data:/app/data
-    ports:
-      - "3001:3001"
-    restart: unless-stopped
-
   ddd-autohaus:
     image: ghcr.io/dqualizer/dqsamples-ddd-autohaus
+    container_name: 01_sample-application
     ports:
-      - "9000:9000"
+      - "${SAMPLE_APPLICATION_PORT}:${SAMPLE_APPLICATION_PORT}"
 
-  dqanalyzer-frontend:
-    image: ghcr.io/dqualizer/dqanalyzer-frontend
+  dqAnalyzer:
+    image: ghcr.io/dqualizer/dqanalyzer2:main
+    container_name: 02_dqanalyzer
     ports:
-      - "9080:80"
+      - "${DQANALYZER_PORT}:80"
```

Comment:
By previous convention, the ports specified in env should be the **outside** available ports. Keep consistent with that or explicitly rename the variable. Also check all other occurrences!

Reply:
Host:Container 
In this case, the variable is available outside and 80 is the internal port.
Should be fine?

Reply:
True. I always get confused by the order...